lib/oca-epak/oep/client.rb

```diff
@@ -20,6 +20,21 @@ def get_html_de_etiquetas_por_orden_or_numero_envio(opts = {})
         response = client.call(method, message: opts)
         parse_result(response, method)
       end
+
+      # Returns the PDF (Base64 encoded) for a label
+      #
+      # @param [Hash] opts
+      # @option opts [Integer] :id_orden_retiro
+      # @option opts [String] :nro_envio
+      # @return [String] PDF data Base64 encoded
+      def get_pdf_de_etiquetas_por_orden_or_numero_envio(opts = {})
+        method = :get_pdf_de_etiquetas_por_orden_or_numero_envio
+        opts = { "idOrdenRetiro" => opts[:id_orden_retiro],
+                 "nroEnvio" => opts[:nro_envio],
+                 "logisticaInversa" => false }
```

Comment:
@cavi21 Shouldn't `logisticaInversa` be an option inside `opts`? It seems that this is associated with the possibility of order "being returned" (devoluciones) 

You could make `false` the default.